web/src/components/series.astro

```diff
@@ -0,0 +1,463 @@
+---
+const gh_branch = import.meta.env.PUBLIC_GH_BRANCH;
+
+const series_100 = {
+  id: "100",
+  title: "FLARE 100: Core",
+  subseries_list: [
+    {
+      id: "101",
+      title: "101: Python APIs",
+      description: "Learn to use the Client API, Model Controller API, and Job API.",
+      cards: [
+        {
+          title: "Getting Started",
+          tags: ["beg.", "pytorch", "lightning", "sklearn", "tensorflow"],
+          description: "Getting started examples using the Client API, Model Controller API, and Job API for different frameworks.",
+          link: `https://github.com/NVIDIA/NVFlare/tree/${gh_branch}/examples/getting_started`
+        },
+        {
+          title: "Client API",
+          tags: ["docs"],
+          description: "The FLARE Client API provides an easy way for users to convert their centralized, local training code into federated learning code.",
+          link: "https://nvflare.readthedocs.io/en/main/programming_guide/execution_api_type/client_api.html#client-api"
```

Comment:
This link to main , what do we do about this?

Reply:
good catch, I'll submit a follow up PR for all the doc links